specification/powerbidedicated/resource-manager/Microsoft.PowerBIdedicated/stable/2021-01-01/powerbidedicated.json

```diff
@@ -857,7 +857,7 @@
           "description": "A collection of Dedicated capacity administrators"
         },
         "mode": {
-          "description": "The capacity mode.",
+          "description": "Indicates the Power BI Embedded capacity generation. 'Gen1' for the legacy generation and 'Gen2' for the latest generation of Power BI Embedded. If no value is specified, the default is Gen2. [Learn More](https://docs.microsoft.com/power-bi/developer/embedded/power-bi-embedded-generation-2)",
```

Comment:
Use one of these strings:

Specifies the generation of the Power BI Embedded capacity. If no value is specified, the default value 'Gen2' is used. [Learn More](/power-bi/developer/embedded/power-bi-embedded-generation-2)

Or

Specifies the generation of the Power BI Embedded capacity. Use 'Gen1' for the legacy Gen1 capacity, and 'Gen2' for the latest Power BI Embedded capacity. If no value is specified, the default value 'Gen2' is used. [Learn More](/power-bi/developer/embedded/power-bi-embedded-generation-2)